packages/flask/src/UICarouselView/CarouselView/Pagination.tsx

```diff
@@ -17,9 +23,18 @@ function Circle({ active, onPress, theme }: CircleProps) {
     const { animatedStyles } = usePaginationStyle(active, theme);
 
     return (
-        <TouchableOpacity hitSlop={UIConstant.carousel.circleHitSlop} onPress={onPress}>
-            <Animated.View style={[styles.circle, animatedStyles]} />
-        </TouchableOpacity>
+        <Animated.View style={[animatedStyles, styles.circle]}>
+            <TouchableOpacity hitSlop={UIConstant.carousel.circleHitSlop} onPress={onPress}>
+                <UIBackgroundView
```

Comment:
Maybe make it also Animated? I guess there is no need to remove animation completely, just separate it to two components

Reply:
@savelichalex it happens so quickly that I can't see the difference with and without color animation. Do we really need animation?

Reply:
It's up to you, as soon as it looks good